deploy/superset/setup-database.sh

```
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Script to add BIM data database connection to Superset via API
+# Run this after Superset is deployed and admin user is created
+
+set -e
+
+SUPERSET_URL="${SUPERSET_URL:-https://superset-production-9a66.up.railway.app}"
+ADMIN_USERNAME="${ADMIN_USERNAME:-admin}"
+ADMIN_PASSWORD="${ADMIN_PASSWORD:-change-me-strong-password}"
+DATABASE_URL="${DATABASE_URL:-${{Postgres.DATABASE_URL}}}"
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)